### DISCUSSION TITLE
Run EAM in an elevated CO2 configuration

### DISCUSSION THREAD

**Discussion by pgoddard08** [E3SM-Project/E3SM#6630] [E3SM model help]
Hello,

I'd like to run an F-case with perpetual CO2 forcing at, let's say 2080 under SSP370, to compare with the F2010 case. I'm not sure where to begin with this task - I appreciate the help.
Paul

**Comment by mahf708** [comment]
It seems like you don't care about the exact numbers, right? What you can do is try to set co2vmr numbers in user_nl_eam (e.g., co2vmr=390.0e-6 in 2010 vs co2vmr=750.0e-6 in 2080 following SSP370). 

To get more help, you could share a run script (e.g., your F2010 case) or more details ... 

You can also set (via xmlchange) the following parameter: CCSM_CO2_PPMV

**Reply by pgoddard08** [reply]
Hi,
Thank you for your reply. I will give that a try.

Could you take a look at my F2010 run script (before adding the co2vmr=###)? I'm unsure what to put in the fields for MODEL_START_TYPE, START_DATE, RUN_REFDIR (if needed, you can point me to a file on NERSC/Perlmutter), RUN_REFCASE, and RUN_REFDATE. They are currently set for a AMIP simulation I ran previously and I'm unsure what to change these to for the F2010 case.
[run.v2.1.LR.F2010_0101_20240923_resubmit_test.txt](https://github.com/user-attachments/files/17101568/run.v2.1.LR.F2010_0101_20240923_resubmit_test.txt)

Thanks.
